compiler/resolution/implementForallIntents.cpp

```diff
@@ -1819,88 +2102,46 @@ static void setShadowVarFlagsNew(Symbol* ovar, VarSymbol* svar, IntentTag intent
 // so an ordered traversal of the two lists gives matching pairs.
 // Count the number of these variables into 'numOuterVars'.
 //
-static void createShadowVarsNew(Expr* refNode, SymbolMap& uses,
-                             int& numShadowVars, int& totOuterVars,
+static void createShadowVarsNew(Expr* refNode, FIIvec& fivec,
```

Comment:
I got to here in my most recent review attempt. I'll have to return to it again.